main.go

```diff
@@ -132,6 +134,7 @@ func init() {
 	flag.BoolVar(&IsExplicitTrust, "x-trust", false, "Explicitly trust Geo Trust CA and Apple IST CA 2 root certificates (Usually you should not need to do this)")
 	flag.StringVar(&DeviceToken, "token", "", "Required. Hexadecimal or Base64 encoded push token for the device")
 	flag.StringVar(&PushTopic, "topic", "", "Required. The topic the device subscribes to")
+	flag.StringVar(&PushType, "type", "", "The value of the apns-push-type. Possible vaules are: alert background location voip complication fileprovider mdm liveactivity'.  Default is 'alert'")
```

Comment:
A minor thing: there's an extra `'` at `liveactivity` that needs to be removed.

Also, I'd suggest separating the possible values with commas for better readability.

Reply:
fixed